redjanym/fcm-bundle/1.1/config/packages/redjan_ym_fcm.yaml

```diff
@@ -0,0 +1,2 @@
+redjan_ym_fcm:
+    firebase_api_key: "12345"
```

Comment:
Let's use an environment variable for this. 

It is very likely you want to have one key for local development and one for production.

Reply:
You are right. Added the environment variable. Pls have a look.